trdg/utils.py

```diff
@@ -20,19 +20,14 @@ def load_dict(lang):
     return lang_dict
 
 
-def load_fonts(lang):
+def load_fonts(folder='fonts/latin'):
     """Load all fonts in the fonts directories
     """
+    fonts = []
+    if folder is not None:
+        for font in os.listdir(folder):
+            if font.split('.')[-1].lower() in ['ttf', 'otf', 'svg', 'eot', 'woff']:
```

Comment:
That will break further down the line since we use `ImageFont.truetype`.

Reply:
you are right ttf and also otf works

Reply:
I didn't know that otf worked, thanks!